R/rvar-print.R

```diff
@@ -0,0 +1,177 @@
+#' Print or format a random variable
+#'
+#' Printing and formatting methods for [`rvar`]s.
+#'
+#' @encoding UTF-8
+#' @param x,object An [`rvar`].
+#' @template args-print-digits
+#' @template args-print-summary
+#' @template args-print-dots
+#' @param color Whether or not to use color when formatting the output. If `TRUE`,
+#' the [pillar::style_num()] functions may be used to produce strings containing
+#' control sequences to produce colored output on the terminal.
+#' @param vec.len Numeric (>= 0) indicating how many 'first few' elements are
+#' displayed of each vector. If `NULL`, defaults to `getOption("str")$vec.len`,
+#' which defaults to 4.
+#'
+#' @details
+#' `print()` and `str()` print out [`rvar`] objects by summarizing each element
+#' in the random variable with either its mean±sd or median±mad, depending on
+#' the value of `summary`. Both functions use the `format()` implementation for
+#' [`rvar`] objects under the hood, which returns a character vector in the
+#' mean±sd or median±mad form.
+#'
+#' @return
+#' For `print()` and `str()`, an invisible version of the input object.
+#'
+#' For `format()`, a character vector of the same dimensions as `x` where each
+#' entry is of the form `"mean±sd"` or `"median±mad"`, depending on the value
```

Comment:
I originally went with the condensed form so that things would chunk better in vector and matrix output, but I see your point. I added the spaces around +/- and made sure there were 2 spaces between elements in vector/matrix/tibble output. I think that works better than what I had before.